app/controllers/integrated/introduce_yourself_controller.rb

```diff
@@ -17,10 +17,21 @@ def update_models
 
     private
 
+    def combine_birthday_fields(data)
+      data[:birthday] = DateTime.new(
+        data.delete(:birthday_year).to_i,
+        data.delete(:birthday_month).to_i,
+        data.delete(:birthday_day).to_i,
+      )
+    end
```

Comment:
At first I was wondering if we could continue to use Rails' built-in `(2i)`, `(3i)` facility. But this is probably more clear. 👍